src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -56,9 +56,17 @@ public void sendConsentOrderMadeConfirmationEmail(CaseDetails caseDetails) {
         sendNotificationEmail(notificationRequestMapper.createNotificationRequestForAppSolicitor(caseDetails), uri);
     }
 
-    public void sendConsentOrderNotApprovedEmail(CaseDetails caseDetails) {
+    public void sendConsentOrderNotApprovedEmailToApplicantSolicitor(CaseDetails caseDetails) {
+        sendConsentOrderNotApprovedEmail(notificationRequestMapper.createNotificationRequestForAppSolicitor(caseDetails));
+    }
+
+    public void sendConsentOrderNotApprovedEmailToRespondentSolicitor(CaseDetails caseDetails) {
+        sendConsentOrderNotApprovedEmail(notificationRequestMapper.createNotificationRequestForRespSolicitor(caseDetails));
```

Comment:
I assume the name of the CCD fields are the same between consented and contested which allows us to use this method? or is there logic to determine where it should come from in CCD based on the case type?

Reply:
It is calling a "consented" method, so need for other logic I think.